src/app/youTube/pages/details/details.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="flex details-block">
+  <div class="back-button" (click)="backClickHandler()">
+      <img src="assets/images/backIcon.png">
+  </div>
+  <div class="flex shadow">
+    <img class="image" src="{{ this.item.snippet.thumbnails.standard.url }}" />
```

Comment:
in this case it is better to do
`[src]="this.item.snippet.thumbnails.standard.url"`